model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -91,6 +93,15 @@ public void setAction(final String action) {
         this.action = action;
     }
 
+    @ApiModelProperty(value = "Ordered list of policy sets provided by the requestor")
```

Comment:
- how about getPolicySetsEvaluationOrder as the name ?
- api doc: "This list specifies the order in which policy sets will be evaluated.  Evaluation stops when a policy with matching target is found and the condition returns true, Or all policies are exhausted."